camera_matrix:
- [650.6501771915733, 0.0, 340.0555553374907]
- [0.0, 651.3157214142177, 221.99632654030867]
- [0.0, 0.0, 1.0]
dist_coeffs: [0.24010887064193975, -0.07669636157597086, 0.004470512814208809, 0.01873949399259765, -1.1890071897096737]
overall_rms: 0.37548309658090784
